frontend/src/Pages/EditPage.tsx

```diff
@@ -0,0 +1,106 @@
+import Header from "../components/header/Header";
+import {useNavigate, useParams} from "react-router-dom";
+import {useCallback, useEffect, useState} from "react";
+import {deleteGame, editOneGame, fetchAllGames, getOneGameToEdit, switchGameStatus} from "../service/apiService";
+import {Button, Grid, Switch, TextField} from "@mui/material";
+import {Game} from "../service/model";
+import DeleteForeverIcon from "@mui/icons-material/DeleteForever";
+
+export default function EditPage(){
+
+    const {id} = useParams()
+    const [game, setGame] = useState({} as Game)
+    const [gameName, setGameName] = useState("")
+    const [errorMessage, setErrorMessage] = useState("")
+    const [roles, setRoles] = useState([""])
+    const nav = useNavigate()
+
+    const fetchGame = useCallback(() => {
+        if (id) {
+            getOneGameToEdit(id)
+                .then(response => response.data)
+                .then(data => {
+                    setGame(data)
+                    setGameName(data.gameName)
+                })
+                .then(() => setErrorMessage(""))
+                .catch((error) => {
```

Comment:
I think you can set the error message to error.response.data right away without if check.